frontend/lib/main_local.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:ehrenamtskarte/configuration.dart';
+import 'package:flutter/material.dart';
+
+import 'app.dart';
+
+Future<void> main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  runApp(Configuration(
+      mapStyleUrl: "http://10.0.2.2:5002/style.json",
```

Comment:
I'm not sure whether this works on all environments. This maybe does not work when using a real device instead of an emulator.

better way would be to use localhost and use adb reverse.

Reply:
Good point, I will change it to localhost then